node/resolvers/search/index.ts

```diff
@@ -464,7 +464,7 @@ export const queries = {
         products = await search.productsByReference(values, salesChannel)
         break
       case 'sku':
-        products = await search.productBySku(values, salesChannel)
+        products = await search.productsBySku(values, salesChannel)
```

Comment:
The returned objects from `productBySku` and `productsBySku` are different, aren't they? Could this be an issue?

Reply:
They are the same return value, it's an array just like the other methods:

```
products = await search.productByReference(value)
```